resource/src/routes/route.mustache

```diff
@@ -0,0 +1,57 @@
+const { Router } = require('express');
+const {{camelCasePlural}} = require('../controllers/{{snakeCasePlural}}');
+
+const serialize = ({{camelCase}}) => {
+  const data = Object.assign({}, {{camelCase}});
+  {{#isUser}}
+  delete(data.password);
+  {{/isUser}}
+  return data;
+};
+
+const router = module.exports = new Router()
+  .get('', function * (req, res) {
+    const list = yield {{camelCasePlural}}.all(req.query);
+    res.json(list.map(serialize));
+  })
+  .get('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.find(req.params.id);
+    res.json(serialize({{camelCase}}));
+  })
+  .post('', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.create(req.body);
+    res.status(201).json(serialize({{camelCase}}));
+  })
+  .put('/:id', function * (req, res) {
+    const {{camelCase}} = yield {{camelCasePlural}}.update(req.params.id, req.body);
```

Comment:
For PUT vs PATCH I think it's worth changing. It's certainly possible to use it as a defacto patch, but you end up with that weird thing where you have to `foo.bar = null` in order to delete a thing from the database. Except then it's not really deleted, it's null. Blergh.